query-pull/src/lib.rs

```diff
@@ -232,9 +232,7 @@ impl Puller {
         // - Recursing. (TODO: we'll need AttributeCaches to not overwrite in case of recursion! And
         //   ideally not do excess work when some entity/attribute pairs are known.)
         // - Building a structure by walking the pull expression with the caches.
-        //   TODO: aliases.
         // TODO: limits.
-        // TODO: fts.
```

Comment:
I didn't see any obvious FTS extractions in the tests.  Are there some?

Reply:
There are now!